docs/sdk-and-tools/rest-api/transactions.md

```diff
@@ -65,16 +68,16 @@ POST https://gateway.elrond.com/transaction/send HTTP/1.1
 Content-Type: application/json
 
 {
-    "version": 1,
-    "chainId": "v1.0.141",
     "nonce": 42,
     "value": "100000000000000000",
     "receiver": "erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r",
     "sender": "erd1njqj2zggfup4nl83x0nfgqjkjserm7mjyxdx5vzkm8k0gkh40ezqtfz9lg",
     "gasPrice": 1000000000,
     "gasLimit": 70000,
-    "data": "food for cats",
-    "signature": "93207c579bf57be03add632b0e1624a73576eeda8a1687e0fa286f03eb1a17ffb125ccdb008a264c402f074a360442c7a034e237679322f62268b614e926d10f"
+    "data": "Zm9vZCBmb3IgY2F0cw==", #base64 representation of "food for cats"
+    "signature": "93207c579bf57be03add632b0e1624a73576eeda8a1687e0fa286f03eb1a17ffb125ccdb008a264c402f074a360442c7a034e237679322f62268b614e926d10f",
```

Comment:
Didn't check the signature, might be incorrect - but can be left as it is for now.

Reply:
surely it doesn't match anymore, but it is just an example and one can see it's size and format, so it only acts as a placeholder